notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>I'm not familiar with the definition or idea behind "Minimum Recommended Products." Can you briefly explain, and maybe include a link to any documentation that explains more? (From astroquery, or MAST?)</p><p> </p><p>So it looks like data_products would have a large list of products, and we can specify that they include raw frames, calibrations, blah blah whatever else. But only one row in the list will be the Minimum Recommended Product, is that right? Could there be more than one? (In which case the "data" call below would download multiple products?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>